src/utils.php

```diff
@@ -7,6 +7,7 @@
 use InvalidArgumentException;
 use PhpParser\Node;
 use Sabre\Event\{Loop, Promise, EmitterInterface};
+use function Sabre\Uri\parse;
```

Comment:
please only alias `Sabre\Uri` and reference with `Uri\parse()`, otherwise it is harder to understand what is going on.